README.md

```diff
@@ -59,8 +58,16 @@ or install top level dataset by omitting `-r datasets` option and then calling
 
 You can navigate datasets you have installed in your terminal or browser,
 while fetching necessary files or installing new sub-datasets using the
-same `datalad install [FILENAMES]` command.
-
+`datalad get [FILE|DIR]` command.  DataLad will take care about
```

Comment:
Additionally (1.): Now, we install several subdatasets with `recursive` and pass `recursion_limit`. Will `recursion_limit` be anchored at the toplevel we are calling `install` from or at each of the subdatasets, which might resist at different hierarchy levels?